webui/api-ts-sdk/Makefile

```diff
@@ -35,7 +35,4 @@ build: build-swagger
 
 .PHONY: clean
 clean:
-	rm -r deps/ || true
```

Comment:
**question:** genuinely curious - if `deps` doesn't exist, then the `rm` fails but the `|| true` allows the next command to run without failure, but would the error get printed to the stdout? Why choose `rm -r X || true` over `rm -rf X` I guess is the real question?

Reply:
I had to lookup -f again the man page shows:
```      
   -f, --force
                ignore nonexistent files and arguments, never prompt
```

I think the difference would be in the never prompt portion.. or less likely if you don't have `rm` or mistype something

![Screen Capture_select-area_20200723121827](https://user-images.githubusercontent.com/12127420/88329439-cf3e6400-ccde-11ea-8ca2-4d2b4b016f36.png)